Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectBuilder/MSBuildProjectBuilderEditor.cs

```diff
@@ -89,8 +89,16 @@ void OnUpdate()
             //[MenuItem("MSBuild/Auto Build All Projects [testing only]", priority = int.MaxValue)]
             private static void BuildAllAutoBuiltProjects()
```

Comment:
Should this still be a commented out line?

Reply:
Yes, this is the "auto build" that gets called when you load the Unity project. The menu item is supposed to just be for testing so I don't have to restart Unity to test this code path. There could be some reasonable argument for exposing it as a feature, but I wasn't really considering this for this change.